fastchat/train/train.py

```diff
@@ -117,19 +118,17 @@ def preprocess(
         total_len = int(target.ne(tokenizer.pad_token_id).sum())
 
         turns = conversation.split(conv.sep2)
-        cur_len = 1
-        target[:cur_len] = IGNORE_TOKEN_ID
```

Comment:
@shibing624 The first token is for BOS. I found BOS is dropped after your PR. Could you fix this?